.travis.yml

```diff
@@ -11,50 +11,56 @@ env:
     - PHPUNIT_FLAGS="-v"
     - PHPUNIT_ENABLED="true"
     - COVERALLS_ENABLED="false"
+    - IMAGICK_ENABLED="true"
+    - ENQUEUE_ENABLED="true"
     - STABILITY=stable
 
 matrix:
   fast_finish: true
   include:
     - php: '7.1'
     - php: '7.2'
+    - php: '7.3'
+    - php: '7.4'
 
     # Enable code coverage with the latest supported PHP version
-    - php: '7.3'
+    - php: '8.0'
       env:
         - COVERALLS_ENABLED="true"
+        - IMAGICK_ENABLED="false"
+        - ENQUEUE_ENABLED="false"
         - XDEBUG_MODE=coverage
         - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
 
     # Minimum supported dependencies with the latest and oldest supported PHP versions
     - php: '7.1'
       env:
         - COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.3'
+    - php: '8.0'
```

Comment:
This is the last failing job. I think we should stick with php 7.4 here as having php 8.0 with --prefer-lowest would work only if any dependencies doesn't declare supporting php with the `>=` operator without being compatible with php 8.0.

Note: Symfony does the same (see https://github.com/symfony/symfony/blob/5.x/.travis.yml#L23-L30)

Reply:
yeah, i agree. we could also drop the "latest php version prefer-lowest" completely. the main thing we care about is "lowest php version with prefer-lowest".

Reply:
The highest supported PHP version with lowest dependencies tests the case where, in a project, an unrelated dependency lock one of our dependency to a lower version than expected.

I took the idea from Symfony IIRC but maybe this is already covered with other jobs. I’m neutral about keep/remove this job from the matrix.

Reply:
there is always a chance of some combinations of versions having a problem. but builds are fast and parallel, so lets change this to 7.4 for now.